README.md

```diff
@@ -167,6 +182,12 @@ Before running `3scale API publishing Pipeline (publish-api-3scale)`, please rea
 
 2) In Jenkins 2, running script in pipeline are subject to script security check (this pipeline was developed with Groovy script), and the script has to be approved before it is allowed to run.  When you start to run `3scale API publishing Pipeline`, you mostly will encounter "Scripts not permitted to use method ... " exceptions. If this happens, please login Jenkins as an administrator, go to "Manage Jenkins", "In-process Script Approval" page to approval the script.  You may need to do the approval multiple times as we found that Jenkins does the check on method level.  You only need to approve once on all those methods during the first run.  We suggest you to use persistent storage enabled Jenkins pod so that your approvals can be saved and reloaded if pod is restarted.
 
+3) Create the routes for your APIcast gateways in 3scale Project if required with below command
+
+```sh
+oc new-app -f apicast-routes-template.yaml -p BASE_NAME=3scalefuse -p WILDCARD_DOMAIN=app.rhdp.ocp.cloud.lab.eng.bos.redhat.com -n ah-3scale-ansible
```

Comment:
app.rhdp.ocp.cloud.lab.eng.bos.redhat.com <- would this expose Red Hat internal infrastructure?

Reply:
change done.